jekyll/_plugins/api_endpoint.rb

```diff
@@ -45,12 +45,12 @@ def curl_args_padded(endpoint)
     def curl_params(endpoint)
       params = endpoint['params']
       if params and (endpoint['method'] == 'GET')
-        '&' + params.map {|param| "#{param['name']}=#{param['example']}"}.join('&')
+        '?' + params.map {|param| "#{param['name']}=#{param['example']}"}.join('&')
       end
     end
 
     def api_curl(endpoint)
-      "curl #{curl_args_padded(endpoint)}https://circleci.com#{endpoint['url']}?circle-token=:token#{curl_params(endpoint)}"
+      "curl -u <circle-token>: #{curl_args_padded(endpoint)}https://circleci.com#{endpoint['url']}#{curl_params(endpoint)}"
```

Comment:
Do we prefer customers use this over `-H Circle-Token:...`? Both seem fine, just curious which is considered the ideal way to authenticate, if we have already chosen one.

Reply:
Our API v2 docs, seem to recommend using headers. 

> Authentication
> API Key (api_key_query)
>
> Parameter Name: circle-token, in: query.
> API Key (api_key_header)
>
> Parameter Name: Circle-Token, in: header.
>HTTP Authentication, scheme: basic HTTP basic authentication. The username should be set as the circle-token value, and the password should be left blank. Note that project tokens are currently not supported on API v2.